public/sass/components/_dashboard_grid.scss

```diff
@@ -13,6 +13,8 @@
   &:hover {
     .react-resizable-handle {
       visibility: visible;
+      // above other panels, but below lowest pre-defined index in zIndex.ts
+      z-index: 999;
```

Comment:
Do you think we would benefit from defining this `z-index` value in `zIndex.ts`? It seems that the value is context-dependent, and I am concerned that if that file changes its values, this element will no longer be below the lowest pre-defined index.

Reply:
i wanted to put it there, but how do we import it into scss files? 🤔

Reply:
🤔 that's a great question, it seems we already do that in other places https://github.com/grafana/grafana/blob/main/public/sass/components/_dashboard_settings.scss#L9, we are mapping them here, but tbh I am not very familiar with the whole themes https://github.com/grafana/grafana/blob/6614eb0a6e8cf2ccce55bec2d99bf4e33601cf46/packages/grafana-ui/src/themes/_variables.scss.tmpl.ts#L164

Reply:
ah, nice. i'll do that then 👍